library/modules/EventManager.cpp

```diff
@@ -1044,7 +1029,7 @@ static void manageUnitAttackEvent(color_ostream& out) {
 }
 
 static std::string getVerb(df::unit* unit, std::string reportStr) {
-    std::string result(reportStr);
+    std::string result(std::move(reportStr));
```

Comment:
Ok, I'll do the thing.

Reply:
The latest commit doesn't really address my comment. It wasn't std::move that I was objecting to, it was the initialization of a variable that was just being overwritten in every scope that it was used in. However, what you have is better than what was there, and I'm not going to hold this PR up over this nit.

Reply:
Oh I see.

Well every place it was used it gets returned after it's done being manipulated. So it wouldn't change much. I think it might actually just be a larger file size compiled if declared in each scope separately.. no?